include/wx/typeinfo.h

```diff
@@ -127,10 +115,10 @@ class wxTypeIdentifier
         #endif // __VISUALC__/!__VISUALC__
     };
 
-    wxTypeIdentifier(const char* ptr) : m_ptr{ptr} { }
+    wxTypeIdentifier(const char* ptr) : m_ptr(ptr) { }
```

Comment:
Not sure why this was changed? It's not a big deal, of course, but using `{}` to initialize this member looked fine.

Reply:
It was mostly done for consistency and for no other reason.